publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -406,51 +429,111 @@ void shouldReturnUnauthorizedWhenUserCannotBeIdentified() throws IOException {
         assertThat(response.getStatusCode(), is(equalTo(HttpURLConnection.HTTP_UNAUTHORIZED)));
     }
 
+    @Test
+    void shouldUpdateForeignResourceWhenAuthorizedUserIsContributorAndHasCristinId()
```

Comment:
Not sure if "foreign" is the right description here. Suggestion: shouldUpdateResourceWhenAuthorizedUserIsContributorAndHasCristinId